app/components/screen/screen.tsx

```diff
@@ -16,6 +16,12 @@ const useStyles = makeStyles(({ colors }) => ({
   background: {
     color: colors.white,
   },
+  notificationbarlight: {
+    color: colors._white,
```

Comment:
You should use .white instead of ._white.

It automatically provide color conversion across mode.

This will reduce this PR to only 2 or 3 lines of code